scripts/run_action.sh

```diff
@@ -22,27 +22,41 @@ set -eu
 
 export ROOT_DIR=${ROOT_DIR:-$(pwd)/$(git rev-parse --show-cdup)}
 export BUILD_DIR=${BUILD_DIR:-${ROOT_DIR}/build}
-export _DO_ACTION_INDENT=''
+export RUN_ACTION_INDENT=''
 
-function do_action() {
-  local -r OLD_INDENT="${_DO_ACTION_INDENT}"
-  _DO_ACTION_INDENT="..${_DO_ACTION_INDENT}"
+function run_action() {
+  local -r ACTION="${1:-""}"
   local -r STYLE_BOLD_WHITE='\033[1;37m'
+  local -r STYLE_BOLD_GREEN='\033[1;32m'
   local -r STYLE_BOLD_RED='\033[1;31m'
   local -r STYLE_RESET='\033[0m'
-  local -r action="$1"
-  echo -e "${OLD_INDENT}${STYLE_BOLD_WHITE}[Running ${action}]${STYLE_RESET}"
+  local -r OLD_INDENT="${RUN_ACTION_INDENT}"
+
+  RUN_ACTION_INDENT="=> ${RUN_ACTION_INDENT}"
+
+  if [[ "${ACTION}" == "" ]]; then
+    echo -e "Please provide an action to run. ${STYLE_BOLD_WHITE}List of valid actions:${STYLE_RESET}\n"
+    find . -name '*.action.sh' | sed -E 's:./src/(.*).action.sh:\1:'
```

Comment:
I think the dots in the `sed` script are being interpreted as wildcards, so this is probably not doing exactly what you mean.  If it's easy to escape them, that would probably be better.

Reply:
Ah, great point. Fixing